index.js

```diff
@@ -166,15 +164,11 @@ module.exports = {
       });
     } else {
       isPercyEnabled = false;
-
-      if (environment.ci && !token) {
+      console.warn('[percy] Percy is not running.');
```

Comment:
We should silently support running in local environments without Percy enabled, it shouldn't be a warning (or probably any sort of notification at all). There are lots of cases where you'd run test cases or suites without Percy turned on, it should probably remain silent.

Reply:
Turns out we don't have a pattern of having debug messages in the javascript clients, only for ruby. I'm going to remove this instead of creating a one-off debug message for something that isn't very important.